labs/spring_2016_update.md

```diff
@@ -0,0 +1,29 @@
+Subject: You Are Now Hack Club's Ambassadors
```

Comment:
What if we changed the subject to "Hack Club Update" or "Hack Club Semester 2 Update"? I don't think we're turning our Labs club leaders into "ambassadors" with this email.